### DISCUSSION TITLE
PHP 8.2 dynamic property deprecation

### DISCUSSION THREAD

**Discussion by npo-mmenke** [spatie/once#98] [Bugs]
Hey, are there any plans to adress the deprecations which are triggered in 8.2+?

Creation of dynamic property ...::$___once_listener__451841 is deprecated
Creation of dynamic property Spatie\Once\Listener::$objectHash is deprecated

**Comment by freekmurze** [comment]
Feel free to send a PR to fix this.

**Comment by npo-mmenke** [comment]
nevermind, my bad, this is fixed in the 3.0 release
